Serverless/EventBridgeLambdaAndS3Example/aws-lambda-tools-defaults.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Information": [
+    "This file provides default values for the deployment wizard inside Visual Studio and the AWS Lambda commands added to the .NET Core CLI.",
```

Comment:
Not connected to this review as such, but we should get the service team to drop 'Core' here in the template.